### ISSUE TITLE
Documentation - Add Baro and Vel PID tuning guide.

### ISSUE THREAD

**Issue by l2smith2** [cleanflight/cleanflight#203]
Attempting to tune cleanflight altitude hold in the usual multiwii way doesn't seem to work in the same way. How does the new system work that includes the VEL PID and how do the values effect each other. In lamans terms preferably

**Comment by hydra** [comment]
Lux on freenode irc is a good source of information and may be able to help.

**Comment by jeffridj** [comment]
I've got a afromini 32 which I've been using in a 230 size quad for about a year, but never bothered with altitude hold as I didn't really need it. Now I wanted to do some video and alt hold seemed a good idea. I upgraded to firmware 1.90 and tried all values of P from 5 to 25.5 but nothing seemed to make any difference. I looked at the barometer graph output and saw small changes if I moved the quad up and down rapidly but ZERO changes if I moved the quad up and down slowly. This ties in with what I see when flying it, no sudden height changes but doesn't hold a set height at mid throttle. To prove the barometer is OK I put baseflight on and the graph produced the expected output when moving the quad up and down. In a test flight on default settings the alt hold was good but not perfect. Can anyone suggest what I am doing wrong as I would prefer to keep cleanflight?
